apps/palette-generator/paletteJsonGenerator.ts

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright 2024 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+import { v4 as uuidv4 } from "uuid";
+
+const BLANK_CELL_LABEL = "BLANK";
+const SVG_PREFIX = "SVG:";
+const SVG_SUFFIX = ":SVG";
+const LABEL_MARKER = "LABEL:";
```

Comment:
When I input this line in the "Glosses" text area:

```
BLANK 18267LABEL:aaa induction
```
to alter the label for a symbol. I got this error in the error section: "BciAvId not found for label: induction". Moving the "LABEL:" marker to the last "induction" gloss reports the same error.

Reply:
That string specifies three cells:  a blank one, one for a plus sign labelled "aaa" and a cell whose gloss is "induction" or contains the word induction.  "Induction" does not match any gloss and that is why there is an error.  Checking blissary.com with "induction" gives "No results found."  The result is as expected.  Moving the LABEL marker won't change the result.

There was some text in the README explaining that spaces are used as a separator to indicate the string to use for a cell.  I've added more warnings to that effect including that LABEL: strings cannot contain spaces and to use an underscore if a space is desired.  Specifically, if the label for the plus sign was supposed to be "aaa induction", then you write it this way:

`BLANK 18267LABEL:aaa_induction`

That gives two cells, a blank and a plus with the label "aaa induction".

Reply:
You are right. The error is reported correctly. I mistakenly thought it was related to the "LABEL" marker.